src/stabilization_guide.md

```diff
@@ -0,0 +1,179 @@
+
+If there was documentation there, integrating it into the
+existing documentation is needed.
+
+If there wasn't documentation there, it needs to be added.
+
+Places that may need updated documentation:
+
+- [The Reference]: This must be updated, in full detail.
+- [The Book]: This may or may not need updating, depends.
+    If you're not sure, please open an issue on this repository
+    and it can be discussed.
+- standard library documentation: As needed. Language features
+    often don't need this, but if it's a feature that changes
+    how good examples are written, such as when `?` was added
+    to the language, updating examples is important.
+- [Rust by Example]: As needed.
+
+Prepare PRs to update documentations invovling this new feature
+for  repositories mentioned above.Maintainers of these repositories
+will keep these PRs open until the whole stabilization process
+has completed. Meanwhile, we can proceed to the next step.
+
+## Write a stabilization report
+
+Find the tracking issue of the feature, and create a short
+stabilization report. Essentially this would be a brief summary
+of the feature plus some links to test cases showing it works
+as expected, along with a list of edge cases that came up and
+and were considered. This is a minimal "due diligence" that
+we do before stabilizing.
+
+The report should contain:
+
+- A summary, showing examples (e.g. code snippets) what is
+  enabled by this feature.
+- Links to test cases in our test suite regarding this feature
+  and describe the feature's behavior on encountering edge cases.
+- Links to the documentations (the PRs we have made in the
+  previous steps).
+- Any other relevant information(Examples of such reports can
+  be found in rust-lang/rust#44494 and rust-lang/rust#28237).
+- The resolutions of any unresolved questions if the stabilization
+  is for an RFC.
+
+## FCP
+
+If any member of the team responsible for tracking this
+feature agrees with stabilizing this feature, they will
+start the FCP (final-comment-period) process by
+commenting
+
+```bash
+@rfcbot fcp merge
+```
+
+The rest of the team members will review the proposal. If the final
+decision is to stabilize, we proceed to do the actual code modification.
+
+## Stabilization PR
+
+Once we have decided to stabilize a feature, we need to have a PR that
+actually makes that stabilization happen. These kinds of PRs are a
+great way to get involved in Rust, as they take you on a little tour
+through the source code.
+
+Here is a general guide to how to stabilize a feature -- every feature
+is different, of course, so some features may require steps beyond 
+what this guide talks about.
+
+Note: Before we stabilize any feature, it's the rule that it should appear
+in the documentation.
+
+### Updating the feature-gate listing
+
+There is a central listing of feature-gates in
+`[src/libsyntax/feature_gate.rs]`. Search for the `declare_features!`
```

Comment:
```suggestion
[`src/libsyntax/feature_gate.rs`]. Search for the `declare_features!`
```

Also, you will probably need to update the definition at the bottom of the target.